pkg/cmd/controller/csr.go

```diff
@@ -0,0 +1,51 @@
+package controller
+
+import (
+	"context"
+	"github.com/openshift/cluster-policy-controller/pkg/operator"
+	"github.com/openshift/library-go/pkg/operator/csr"
+	"github.com/openshift/library-go/pkg/operator/events"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/selection"
```

Comment:
nit: order this such that it's
```
system libs

k8s libs

openshift libs

local libs
```

Reply:
Let's continue this discussion here: https://github.com/openshift/cluster-policy-controller/pull/61#discussion_r654274660